### PR TITLE
updated stache and surrounding docs

### PR DESCRIPTION
**Pull request by Blackbaud-BrandonJones**

### CODE REVIEW

File: src/app/support/changelog/index.html [hunk 1]
```diff
@@ -1,65 +1,30 @@
...
-<stache-page-anchor>
-  2.0.0-rc.0 (2017-07-26)
-</stache-page-anchor>
+  <stache-page-anchor>
+     2.0.0-rc.2 (2017-08-11)
+  </stache-page-anchor>
   <ul>
     <li>
-      Converted `StachePageAnchorComponent` to an anchor tag, to allow users to open the link in a new window. <a href="https://github.com/blackbaud/stache2/pull/277">[#277]</a>
+      Added support for <stache-code>stache-code-block's</stache-code> in <stache-code>stache-include's</stache-code> to contain <stache-code>{{ '{' }}</stache-code> and <stache-code>}</stache-code>. <a href="https://github.com/blackbaud/skyux-builder-plugin-stache/pull/13">#13</a>
```
Comment (Blackbaud-LindseyRix) [inline src/app/support/changelog/index.html]: Can you take off the 's from `stache-code-block` and `stache-include`.

File: src/app/learn/update-stache/index.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
         <stache-code-block
           languageType="json">
             "dependencies": {
-              "@blackbaud/stache": "2.0.0-rc.1",
+              "@blackbaud/stache": "2.0.0-rc.2",
```
Comment (Blackbaud-LindseyRix) [inline src/app/learn/update-stache/index.html]: We should turn the Stache, SKY UX, SKYUX Builder into global variables that we only have to update in one place.

Comment (Blackbaud-BrandonJones) [inline src/app/learn/update-stache/index.html]: That would be great, except currently we can't use actual stache.global variables inside of code blocks as the code blocks escape and don't run that code.  It's something I want to look into allowing somehow, but this is a one off scenario that I think we might be one of the few who would encounter.

Comment (Blackbaud-LindseyRix) [inline src/app/learn/update-stache/index.html]: DOH! Well, let's remember we want to do that someday.

**Comment by Blackbaud-LindseyRix** [comment]
If you fix the 's on the changelog, i'll approve.

File: src/app/learn/update-stache/index.html (hunk 2 above)
Comment (Blackbaud-BrandonJones) [inline src/app/learn/update-stache/index.html]: 👍

**Pull request merged**